main/guides/orchestration/dapp-orchestration-basics/installation/index.md

```diff
@@ -106,6 +109,19 @@ If you need to restart your environment for any reason, use the following comman
 make teardown ; make stop; make stop-forward; make clean; make; make port-forward
 ```
 
+## Troubleshooting Docker
+During setting up the environment, you may run into a few issues with docker. Here are two common setup requirements.
+
+### Disable Kubernetes
+Inside of "settings", navigate to the "kubernetes" section. Here, you want to ensure that you deselect the "Enable Kubernetes" checkbox
+<img src="../../assets/docker-kube.png" width="100%" />
+
+### Increasing resources allocated to docker
+From inside of "settings", navigate to the "Resources" section. By default docker doesn't optimize the resources allocated for running resource-intensive environments. For this environment, we want to ensure our `CPU limit`, `Memory Limit`, and `Swap` are all bumped up to 75% their max. You can always increase/decrease after you get everything setup, and determine how much resources are needed for your machine.
```

Comment:
I think the % depends on your machine. For me, 10GB memory worked, while 8GB did not. My swap was at 1GB in both cases.